test/hotspot/jtreg/containers/docker/TestMisc.java

```diff
@@ -123,6 +123,19 @@ private static void checkContainerInfo(OutputAnalyzer out) throws Exception {
         for (String s : expectedToContain) {
             out.shouldContain(s);
         }
+        String str = out.getOutput();
+        if (str.contains("cgroupv1")) {
```

Comment:
An alternative would be to pass a `Metrics` instance (`Metrics.systemMetrics()`) to `checkContainerInfo` and use:
`"cgroupv1".equals(m.getProvider())` here instead.